src/org/loklak/Caretaker.java

```diff
@@ -69,7 +68,7 @@
     public void shutdown() {
         this.shallRun = false;
         this.interrupt();
-        Log.getLog().info("catched caretaker termination signal");
+        DAO.log("catched caretaker termination signal");
```

Comment:
As of now, the line will log as 
```
2017-05-11 12:47:46.772:INFO :org.loklak.data.DAO:CARETAKER: ...
```
After merging this patch, it will be
```
2017-05-11 12:55:55.340:INFO :org.loklak.data.DAO:Thread-38734: ...
```

@vibhcool: But I am not sure about this. Can you check logs on this branch and verify?